ClinGen/clincoded
# Connects to #966. Connects to #1288. Connects to #1290. Connects to #1291. Connects to #1293.

**Steps to test #966:**
1. Go to VCI and use ClinVar ID `139214` at the variant selection interface.
2. Upon page load, proceed to the **Gene-centric** tab.
3. Expect to see 2 new columns – **syn Z** and **mis Z** – in the **ExAC Constraint Scores** table, as well as their respective descriptions.
![image](https://cloud.githubusercontent.com/assets/6956310/22956431/dfbaca0c-f2d6-11e6-9716-eab4648e65d1.png)

**Steps to test #1288:**
1. Create a new GDM in GCI and add an Experimental evidence.
2. On the Experimental curation page, select **Model Systems** or **Rescue** as the Experiment type. Fill out other **required** fields.
3. In the input fields to add HPO IDs (2 for **Model Systems** and 1 for **Rescue**), enter multiple IDs (delimited with comma) and save the form.
4. Expect to see the form to be successfully submitted without errors indicating the limit of 1 HPO ID.
![image](https://cloud.githubusercontent.com/assets/6956310/22956782/e62a348e-f2d8-11e6-925e-7c4a2c02a5c1.png)

**Steps to test #1290:**
1. In the GCI's Create New GDM interface, add an **Orphanet ID** with or without a colon (e.g. **"ORPHA891"** or **"ORPHA:891"**). Fill out other **required** fields. Expect to be able to save the form without errors.
2. On the other curation form pages (e.g. Group, Family, Individual), also expect to be able to enter the **Orphanet ID** with or without a colon (e.g. **"ORPHA891"** or **"ORPHA:891"**) and save without errors.
![image](https://cloud.githubusercontent.com/assets/6956310/22957153/58dc4d1c-f2db-11e6-97b9-db0c194cd9da.png)
3. in the VCI, start a new Interpretation and add an associated disease. In the modal, also expect to be able to enter the **Orphanet ID** with or without a colon (e.g. **"ORPHA891"** or **"ORPHA:891"**) and save without errors.
![image](https://cloud.githubusercontent.com/assets/6956310/22957230/cdd86bf0-f2db-11e6-9a21-92fffe8c17c7.png)

**Steps to test #1291:**
1. While still in the VCI, go to the **Population** tab, expect to see a new **gnomAD** panel with 2 links.
![image](https://cloud.githubusercontent.com/assets/6956310/22911778/59b915fa-f216-11e6-8399-d5a4d59f6d9f.png)
2. Clicking on the first link will take the user to the gnomAD site in the attempt to retrieve/display the variant data (if any).
3. Clicking on the second link will take the user to the gnomAD homepage.

**Steps to test #1293:**
1. Return to the GCI's GDM, go into various curation form pages (e.g. Group, Family, Individual, Case-Control, Experimental), expect to see all instances of HPO IDs input fields have been changed from `text` type to `textarea` type.
![image](https://cloud.githubusercontent.com/assets/6956310/22957472/4e1149f8-f2dd-11e6-898a-b82daa3d7bac.png)
